test/integration/importSuperA-la-carte.spec.js

```diff
@@ -0,0 +1,52 @@
+import Vue from 'vue'
+import VApp from '../../src/components/VApp/VApp'
+import VFooter from '../../src/components/VFooter/VFooter'
+import VContainer from '../../src/components/VGrid/VContainer'
+import VContent from '../../src/components/VGrid/VContent'
+import VNavigationDrawer from '../../src/components/VNavigationDrawer/VNavigationDrawer'
+import VToolbar from '../../src/components/VToolbar/VToolbar'
+import Vuetify from '../../src/components/Vuetify'
+import { mount } from 'avoriaz'
+import { test } from './describe'
+
+import parse from 'posthtml-parser'
+import renderxml from 'posthtml-render'
+
+import { pascalizeTree } from './fixtures/transformTree'
+
+import { compileToFunctions } from 'vue-template-compiler'
+
+const kebabTemplate = `<v-app>
+<v-navigation-drawer app></v-navigation-drawer>
+<v-toolbar app></v-toolbar>
+<v-content>
+  <v-container fluid>
+  </v-container>
+</v-content>
+<v-footer app></v-footer>
+</v-app>`
+
+test('super a-la-carte import', () => {
+  Vue.use(Vuetify, {
+    components: {
+      VApp,
+      VContainer,
+      VContent,
+      VFooter,
+      VNavigationDrawer,
+      VToolbar
+    }
+  })
+  it('should render kebab cpmponents', () => {
```

Comment:
type

Reply:
![image](https://user-images.githubusercontent.com/10551234/43040522-9421339c-8d4e-11e8-88f5-d76af64057ae.png)
Thank, I changed.
Tests content will obviously be changed.